### ISSUE TITLE
How is this different than https://github.com/sabinadams/aurora?

### ISSUE THREAD

**Issue by synerp** [armandov91/prisma-schema-generator#1]
Hi there!
How is this version different from Sabin's version?

Thanks

**Comment by armandov91** [comment]
Hi there!

The main difference is that the aurora.config.json that sabin's version use is automatically generated by this package... so imagine a complex structure of folders... you do not need to add them manually on the aurora.config.json file.

Everytime you execute the "psgen" command it will update the file for you regardles of the hierarchy of folder or files that you have in the "schema's folder"

Basically is mostly an enhance to sabin's work... I believe this should be added directly to his package so that aurora could handle this directly.

**Comment by synerp** [comment]
Ah I see.  This one generates one schema from a folder structure of fragments while Aurora can create multiple schemas from a selection of fragments defined in aurora.config.json.

I use Aurora to generate a dozen different schemas merging different models.  Aurora has some really weird bugs where the order of the models in aurora.config.json changes the end schema.   For example it would drop @@unique when merging two models.

I might try to see if yours works using symlinks to the model.
